src/website/views/utilities/close.pug

```diff
@@ -0,0 +1,40 @@
+---
+title: Close
+---
+
+p.-text Use a dismiss button with the class <code>-close</code> to use in components who need to hide the information
+  |  ( alerts, modals, etc).
+
+.example.-mb3
```

Comment:
@jllr What are your thoughts on documenting sizes here? Since it's a button, it respects `-small` and `-large` button modifiers